sessionctx/variable/session.go

```diff
@@ -644,6 +645,14 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		return ErrReadOnly
 	case TiDBMaxChunkSize:
 		s.MaxChunkSize = tidbOptPositiveInt32(val, DefMaxChunkSize)
+		if s.InitChunkSize > s.MaxChunkSize {
```

Comment:
I think it is better to return an error here.

Reply:
return error in here will break back compatibility, for example: most of our test case that attend to set chunk size will fail.

Reply:
It's weird, people set an invalid `tidb_max_chunk_size` then `tidb_init_chunk_size` is changed...

How about setting it to max(DefMaxChunkSize, tidb_init_chunk_size) and return a warning?